### PR TITLE
Animal page responsive #280

### PR DESCRIPTION
**Pull request by Riddhi162**
## Description
[Please include a brief description of the changes or features added]
I have made the page responsive also improved the font positioning and size etc.

## Related Issue
[Cite any related issue(s) this pull request addresses. If none, simply state "None"]

## Screenshots / GIFs (if applicable)
[Attach any relevant screenshots or GIFs demonstrating the changes]
Screenshot 2024-05-19 122923
Screenshot 2024-05-19 122929
Screenshot 2024-05-19 122759

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

File: src/pages/user/ReportIncidentPages/AnimalDetailsPage/AnimalDetailsPage.jsx [hunk 1]
```diff
@@ -101,10 +101,11 @@ function AnimalDetailsPage({
   }, [formData.predictedAnimal]);
   return (
     // A MASTER CONTAINER
-    <div>
+    <div className="">
```
Comment (codewarnab) [inline src/pages/user/ReportIncidentPages/AnimalDetailsPage/AnimalDetailsPage.jsx:104]: whats this empty classname for ?

File: src/pages/user/ReportIncidentPages/AnimalDetailsPage/AnimalDetailsPage.jsx [hunk 2]
```diff
@@ -443,51 +477,65 @@ function AnimalDetailsPage({
             <div className="bg-white bg-opacity-57 shadow-lg ring-1 ring-gray-300 flex flex-col gap-2 justify-center border-1 p-5 rounded-3xl backdrop-blur-[6px]">
               <label>
                 {" "}
-                <p className="font-extrabold  text-indigo-900 pl-5 text-[1.4rem] leading-normal tracking-wider">
-                  Describe Severity:
+
```
Comment (codewarnab) [inline src/pages/user/ReportIncidentPages/AnimalDetailsPage/AnimalDetailsPage.jsx:480]: change it to describe severity

**Review by codewarnab — CHANGES_REQUESTED** [review]

**Comment by codewarnab** [comment]
@Riddhi162  you have to create two separate pr from two separate brances  for the two pages contact information and animal details page 
also dont forget to mention the issue in decsription

**Comment by Riddhi162** [comment]
ohhh ok ill make the necessary changes.

**Comment by codewarnab** [comment]
this one is the old version of #416 right ? if yes please close the pr

**Comment by Riddhi162** [comment]
ok will do so

**Pull request closed without merging**